### PR TITLE
Update from auto-update/renku-core-2.4.1

### PR DESCRIPTION
**Pull request by RenkuBot**
/deploy extra-values=global.renku.cli_version=2.4.1rc3 #persist

### CODE REVIEW

**Review by rokroskar — CHANGES_REQUESTED** [review]
Thanks @Panaetius! just some minor fixes

**Review by rokroskar — DISMISSED** [review]

**Review by aledegano — DISMISSED** [review]

**Comment by rokroskar** [comment]
@jachro @Panaetius what is this PR waiting for?

**Comment by Panaetius** [comment]
for https://github.com/SwissDataScienceCenter/renku/pull/3083

Also had to create https://github.com/SwissDataScienceCenter/terraform-renku/pull/1409 to be merged after this is merged so Dev works properly

**Comment by Panaetius** [comment]
I'd also like to get https://github.com/SwissDataScienceCenter/renku-actions/pull/49 done first so PR deployments still work without setting the `cli_version` value manually

**Review by rokroskar — DISMISSED** [review]

File: CHANGELOG.rst [hunk 1]
```diff
@@ -1,5 +1,45 @@
 .. _changelog:
 
+0.25.7
```
Comment (aledegano) [inline CHANGELOG.rst:3]: This should be `0.26.0`, we are adding a significant amount of features here, it's not just a patch release.

**Review by rokroskar — APPROVED** [review]

**Review by aledegano — APPROVED** [review]

**Pull request merged**